src/parser/dash-manifest-parser.js

```diff
@@ -50,6 +56,13 @@ class DashManifestParser {
   }
 
   _parseAdaptionSets = () => {
+    const baseURL = XmlUtils.findElements(this._xmlDoc, MpdUtils.TagTypes.BASE_URL);
+    if (baseURL && baseURL.length > 0) {
```

Comment:
looks like it always returns an array
https://developer.mozilla.org/en-US/docs/Web/API/Element/getElementsByTagName

Reply:
@SivanA-Kaltura  what is required here?